package.json

```diff
@@ -31,7 +31,8 @@
     "start:browser": "npm-run-all --parallel 'start:browser-*'",
     "start:browser-backend": "yarn workspace loot-core watch:browser",
     "start:browser-frontend": "yarn workspace @actual-app/web start:browser",
-    "build:server": "yarn build:browser",
+    "build:server": "yarn build:browser && yarn build:sync-server",
+    "build:sync-server": "yarn workspace @actual-app/sync-server build",
```

Comment:
If we don't need two commands it would probably be easier for users to have a single `build:server` defined as `yarn build:browser && yarn workspace @actual-all/sync-server build`